library/src/main/java/com/liulishuo/filedownloader/download/FetchDataTask.java

```diff
@@ -116,7 +116,8 @@ public void run() throws IOException, IllegalAccessException, IllegalArgumentExc
 
         try {
             final boolean isSupportSeek = CustomComponentHolder.getImpl().isSupportSeek();
-            if (hostRunnable != null && !isSupportSeek) {
+            final boolean isMultiConnection = connectionIndex >= 0;
```

Comment:
if it is single connection download the `hostRunnable` must be `null`

Reply:
But in fact, the hostRunnable always is not null.